wezterm-gui/src/termwindow/mod.rs

```diff
@@ -541,10 +538,6 @@ impl TermWindow {
         // force cursor to be repainted
         window.invalidate();
 
-        if let Some(pane) = self.get_active_pane_or_overlay() {
-            pane.focus_changed(focused);
-        }
-
```

Comment:
removing this logic is probably a regression?

Reply:
reverted.  (sorry, I should have caught this).